source/composeWith.js

```diff
@@ -5,7 +5,7 @@ import reverse from './reverse.js';
 
 /**
  * Performs right-to-left function composition using transforming function. The last function may have
- * any arity; the remaining functions must be unary.
+ * any arity; the remaining functions must be unary. Unlike `compose` functions are passed in an array.
```

Comment:
Unlike `compose`, functions are passed in an array.

Add a comma after `compose`, may be more clear?

Reply:
Done.